sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/SparkBatchClient.cs

```diff
@@ -28,5 +31,145 @@ public SparkBatchClient(Uri endpoint, string sparkPoolName, TokenCredential cred
                   options.VersionString)
         {
         }
+
+        /// <summary> Create new spark batch job. </summary>
+        /// <param name="sparkBatchJobOptions"> Livy compatible batch job request payload. </param>
+        /// <param name="detailed"> Optional query param specifying whether detailed response is returned beyond plain livy. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        internal virtual async Task<Response<SparkBatchJob>> CreateSparkBatchJobAsync(SparkBatchJobOptions sparkBatchJobOptions, bool? detailed = null, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("SparkBatchClient.CreateSparkBatchJob");
```

Comment:
nit: typically I've seen something like $"{nameof(SparkBatchClient)}.{nameof(CreateSparkBatchJob)}" to help for refactors.

Reply:
:facepalm: - good idea

Reply:
I'll do this to the non-generated code. I'm going to keep it this way for now on the generated code just to keep it 1:1 for what came out.